test/gtest/ucs/test_memtype_cache.cc

```diff
@@ -233,6 +317,76 @@ UCS_TEST_P(test_memtype_cache, basic) {
     test_ptr_not_found(ptr, size);
 }
 
+UCS_TEST_P(test_memtype_cache, update_non_contig_regions_and_remove_subintervals) {
+    std::vector<test_memtype_cache::region_info> insert_regions;
+    std::vector<test_memtype_cache::region_info> remove_regions;
+    size_t start, end;
+
+    const size_t region_size           = std::pow(2, 28);
+    const size_t interval_start_offset = std::pow(2, 27);
```

Comment:
are these "random" values for start/end/size ?

Reply:
not so random, it is from failing application that uses the same memory ranges